tests/util/s3.py

```diff
@@ -1,7 +1,15 @@
 import boto3
+from botocore.exceptions import ClientError
 
 
-# todo: once able to pass S3 paths as input, remove this ability
+# Downloads input from S3 to local file path.
+# Used for tests that upload local inputs.
 def download_integration_test_input(s3_file_key, output_file_path):
-    s3 = boto3.client('s3')
-    s3.download_file('toolchest-integration-tests', s3_file_key, output_file_path)
+    s3_client = boto3.client('s3')
+    try:
+        s3_client.download_file('toolchest-integration-tests', s3_file_key, output_file_path)
+    except ClientError:
+        try:
+            s3_client.download_file('toolchest-integration-tests-private', s3_file_key, output_file_path)
+        except ClientError as err:
+            raise err
```

Comment:
This is a clever way of trying normal vs. private integration test bucket, but it's pretty ambiguous and not a common pattern.

What about an `is_private` argument that defaults to `False` that can be set manually for the files that are in the private bucket? (or similar)